app/extractors/meck_pomm_pdf_extractor.rb

```diff
@@ -0,0 +1,22 @@
+class MeckPommPDFExtractor
+  def initialize(paper)
+    @contents = paper.contents
+  end
+
+  # Der Minister für Bildung, Wissenschaft und Kultur hat namens der Landesregierung die Kleine Anfrage mit
+
+  ANSWERERS = /\s*(?:Der|Die)\s+(Minister.+?.*)(?:\shat\s+namens?\s+der\s+Land(?:es)?regierung\s+die)\s+(?:[kK]leine|[gG]roße)?\s*An/m
```

Comment:
the affected paper is 6/4640 and the extractor test replicates the situation.. what else? Another one would be 6/1332 which too works fine